versioned_docs/version-8.7/self-managed/identity/access-management/access-management-overview.md

```diff
@@ -0,0 +1,29 @@
+---
+id: access-management-overview
+title: Access Management Overview
+sidebar_label: "Overview"
+description: "Access Management and Permissions overview"
+---
+
+With Identity you can manage the access to Camunda 8 using the following concepts:
+
+- APIs represent the different Camunda 8 components (e.g., Operate, Tasklist). Each [APIs defines for itself a set of permissions](#available-permissions) that can be used to control access to it
+- Permissions are [organized in Roles](./manage-permissions.md#managing-permissions-for-roles), which can be assigned to Users either directly or via Groups. Additonally, permissions can be [assigned to your custom Application](./manage-permissions.md#managing-permissions-for-applications) (e.g., job worker)
+- Optionally, you can grant more fine-grined access control to Camunda 8 resources to Users or Groups using [resource authorizations](./resource-authorizations.md)
+
+## Available Permissions
+
+Each API (representing a component) defines for itself a set of permissions that can be used to control access to it. The following permissions are available:
+
+| Component   | API                                | Permissions                                                 | Descriptions                                                                                                                                                                                                                                                                                                                                                                         |
+| ----------- | ---------------------------------- | ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Identity    | `Camunda Identity Resource Server` | `read` <br/> `read:users` <br/> `write`                     | Read access to entire UI <br/> Access only the Users UI and related subpages <br/> Write access entire UI                                                                                                                                                                                                                                                                            |
```

Comment:
All occurrences of "to UI" should be changed to "to the UI" in this table. We use "to the API" similarly here.

Reply:
Sorry for being annoying, but there are a few remaining occurrences of this in the table.